identity-apps-core/apps/authentication-portal/src/main/webapp/fido2-auth.jsp

```diff
@@ -30,6 +34,24 @@
 
 <%
     String authRequest = request.getParameter("data");
+
+    String authAPIURL = application.getInitParameter(Constants.AUTHENTICATION_REST_ENDPOINT_URL);
+
+    if (StringUtils.isBlank(authAPIURL)) {
+        authAPIURL = IdentityUtil.getServerURL("/api/identity/auth/v1.1/", true, true);
+    } else {
+        // Resolve tenant domain for the authentication API URL.
+        authAPIURL = AuthenticationEndpointUtil.resolveTenantDomain(authAPIURL);
+    }
+    if (!authAPIURL.endsWith("/")) {
+        authAPIURL += "/";
+    }
+    authAPIURL += "context/" + request.getParameter("sessionDataKey");
```

Comment:
Shouldn't we encode request params using owasp encoder, otherwise these can lead to XSS vulnerabilities?

Reply:
Resolved via https://github.com/wso2/identity-apps/pull/4156/commits/8968e9e9e80de6f997577d12f3f6222db508b1de